source/documentation/deploying_services/use_a_custom_domain.erb

```diff
@@ -137,15 +137,9 @@ You can associate up to 100 subdomains with a single cdn-route service instance.
     ```
     cf update-service custom-domains-production \
       -c '{"domain": "www.example.com,www.example.net,www.example.org"}'
-    ```
-
-    If you have previously customised the [cookies](#disabling-forwarding-cookies) or the [headers](#forwarding-headers) forwarded by your cdn-route service, you must include those as configuration parameters in the `cf update-service` command. For example:
+      ```
 
-    ```
-    cf update-service custom-domains-production \
-      -c '{"domain": "www.example.com,www.example.net,www.example.org",
-           "cookies": false, "headers": ["Accept", "Authorization"]}'
-    ```
+<%= warning_text('Whenever you update the "domain" configuration parameter you will have to update your DNS records, regardless of whether the value has changed.') %>
```

Comment:
recommend changing "you will have to" to "you must" so it doesn't sound like you have to do this at some unspecified point in the future

Reply:
Force pushed correction, 🙇